### ISSUE TITLE
Makefile doesn't byte-compile Python code

### ISSUE THREAD

**Issue by polyzen** [jwilk/mwic#9]
> it will create .py files in usr/share/mwic/lib/
> that lack the pyo/pyc files and if running as root the imports will
> create untracked files in that directory that will remain forever

**Comment by jwilk** [comment]
Does d39e277e052f6ab618e82cd3f499504b77b17dd3 address your concerns?

**Comment by polyzen** [comment]
I saw those lines but forget I was looking at master.. Also, my build script does use `DESTDIR`:
https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=mwic

**Comment by jwilk** [comment]
I neglected the case when `DESTDIR` is set on the assumption that distros have their own tools for byte-compliation. Perhaps I was being overly optimistic.
Anyway, this should be fixed in a71847f36da25b612a3d8206782ec5afd9cf5458.
Note that this only takes cares care of non-optimized bytecode. If you want “optimized” bytecode too, run `make install` again with `PYTHONOPTIMIZE` set, i.e. something like this:
```diff
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -24,2 +24,3 @@ package() {
   make PREFIX=/usr DESTDIR="$pkgdir" install
+  make PREFIX=/usr DESTDIR="$pkgdir" install PYTHONOPTIMIZE=1
   install -Dm644 -t "$pkgdir"/usr/share/licenses/mwic doc/LICENSE
```

**Comment by polyzen** [comment]
Tried adding a `compileall` line directly to the build script, and things definitely didn't go as planned. A bunch of lines like:

```
*** Error compiling '/usr/lib/python3.7/pyclbr.py'...                                                                
PermissionError: [Errno 13] Permission denied: '/usr/lib/python3.7/__pycache__/pyclbr.cpython-37.pyc.140693525927080'
```

Will tailor one to build from `master` tomorrow.

**Comment by polyzen** [comment]
Looks good to me! Thank you very much.

**Comment by jwilk** [comment]
Fixed in [0.7.8](https://github.com/jwilk/mwic/releases/tag/0.7.8).
